docs/user/overview.md

```diff
@@ -0,0 +1,61 @@
+# Installer Overview #
+
+The OpenShift Installer is designed to help users, ranging from novices to experts, create OpenShift clusters in various environments. By default, the installer acts as an installation wizard, prompting the user for values that it cannot determine on its own and providing reasonable defaults for everything else. For more advanced users, the installer provides facilities for varying levels of customization.
+
+In [supported environments](#supported-environments), the installer is also capable of provisioning the underlying infrastructure for the cluster. It is recommended that most users make use of this functionality in order to avoid having to provision their own infrastructure themselves. In unsupported environments or scenarios in which the created infrastructure is incompatible, the installer can stop short of creating the infrastructure, allowing the user to provision their own infrastructure using the cluster assets generated by the installer.
+
+
+## Cluster Installation Process ##
+
+OpenShift 4 is unique in that its management extends all the way down to the operating system itself. In order to make this possible, every machine boots with a configuration which references resources hosted in the cluster its joining. This allows the cluster to control how new machines should look and allows the cluster to vary that over time as updates are applied. A downside to this approach, however, is that new clusters have no way of starting without external help - every machine in the as-of-yet-created cluster is waiting on the as-of-yet-created cluster.
+
+OpenShift 4 breaks this dependency loop using a sacrificial bootstrap machine. This bootstrap machine is booted with a concrete [Ignition Config][ignition] which describes how to create the cluster. This machine acts as the remote arm of the installer, created in the target environment for the sole purpose of bringing up the rest of the cluster.
```

Comment:
The word "sacrificial" struck me as a bit odd here.  I think it can be either removed, or perhaps replaced with something like "temporary".